crates/optimism/primitives/Cargo.toml

```diff
@@ -50,7 +50,7 @@ arbitrary.workspace = true
 proptest.workspace = true
 
 [features]
-default = ["std"]
+default = ["std", "serde"]
```

Comment:
this is needed because `reth-primitives` enables `reth-primitives-traits/serde` by default, thus `reth-optimism-traits` does not compile unless serde is enabled :/

once we migrate to alloy block types we should be able to get rid of `reth_primitives` dependency and this hach